RFCs/FS-1040-ireadonly.md

```diff
@@ -0,0 +1,56 @@
+# F# RFC FS-1039 - Implement IReadOnly* interfaces throughout FSharp.Core
```

Comment:
Should be FS-1040?

Reply:
Actually, this should be 1041.  I somehow managed to have an off-by-one error in two different directions.

Reply:
Ah yes, not to be confused with the off by two error